src/UniversalRewardsDistributor.sol

```diff
@@ -32,14 +30,14 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
     /// @notice The address that can update the distribution parameters, and freeze a root.
     address public owner;
 
-    /// @notice The addresses that can update the merkle root.
+    /// @notice The addresses that can update the Merkle root.
     mapping(address => bool) public isUpdater;
 
     /// @notice The timelock related to root updates.
     uint256 public timelock;
 
     /// @notice The pending root of the distribution.
-    /// @dev If the pending root is set, the root can be updated after the timelock has expired.
+    /// @dev If the pending root is set, the root can be updated after the timelock has elapsed.
```

Comment:
Because `expired` has the connotation that it is not valid anymore, but it is actually the opposite